Serenity-Website/src/App.jsx

```diff
@@ -19,21 +23,23 @@ function App() {
 
   useEffect (() => {
     // save the user data to storage whenever use state changes
-    localStorage.setItem('user', JSON.stringify(user));
   }, [user]);
```

Comment:
Question, what is this hook used for now?

Reply:
Oh right, that's one thing I didn't update. So, after researching online, the way to get the initial value of user is to retrieve from a cookie named 'user' using the Cookies.get() method. Since my backend uses cookies and not localStorage, I updated the code. As of now, the updateUser function updates the user state variable with a new value newUser. It also sets the updated user value to the 'user' cookie using Cookies.set(). The useEffect hook ensures that the user data is saved to the cookie whenever the state changes.